cityofaustin/janis
# 3245 "Page is a part of this guide" tiles

Fixes: https://github.com/cityofaustin/techstack/issues/3245

# Description



Deployed: https://janis-3245-page-a-part.netlify.com/

# Types of changes

Thought this is a lot, most of it is just the addition of the new "PageIsPartOfContainer" component and the conditions that comes with having different styling for Information and Services Pages. 

However, I've added the JS function, and `getAllGuidePagesSections` query in the static-config folder to create a key to let Service and Info pages know what guides they're a part of!

Other than that. I've left GitHub notes in the code to explain specifics.

# How can this be tested?
See issue above for testing criteria. But, here are some direct links implementation and examples.
- Here is a *Service* page that is part of _multiple_ guides: https://janis-3245-page-a-part.netlify.com/en/housing-utilities/recycling-trash-and-compost/recycling-reuse/changes-your-clocks-back/
- Here is a *Information* Pages that is part of a _single_ guide: https://janis-3245-page-a-part.netlify.com/en/housing-utilities/recycling-trash-and-compost/recycling-reuse/cool-places-of-interest-to-see-in-austin/

# Checklist:
